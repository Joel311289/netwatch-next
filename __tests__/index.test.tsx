import { render } from '@testing-library/react';
import HomePage from '@/pages/index';

describe('Index page', () => {
  test('renders page correctly', () => {
    const { container } = render(<HomePage />);

    expect(container).toMatchSnapshot();
  });
});
