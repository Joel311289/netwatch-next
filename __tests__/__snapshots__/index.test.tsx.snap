// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page renders page correctly 1`] = `
<div>
  <div
    class="wrapper hasMargins"
  >
    <div
      class="wrapper"
      style="margin-top: 30px;"
    >
      <h2
        class="wrapper"
      >
        welcome.title
      </h2>
      <span
        class="wrapper md"
      >
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Architecto assumenda dolore expedita harum ipsum labore laudantium libero molestias nam numquam odio omnis, quaerat quas rem repellendus similique tempore unde ut.
        <style>
          
        wrapper {
          
        }
      
        </style>
      </span>
      <style>
        
        .wrapper {
          display: flex;
          flex-direction: column;
          flex-wrap: nowrap;
          gap: 10px
          
          
        }
      
      </style>
    </div>
    <h1>
      <a
        href="/movies"
      >
        <button
          class="wrapper medium"
        >
          Movies
        </button>
      </a>
    </h1>
  </div>
</div>
`;
